From: Antonio Valentino <antonio.valentino@tiscali.it>
Date: Sun, 22 Apr 2018 16:32:56 +0000
Subject: remove gtags

Remove Google Analytics script from the doc template layout to
avoid privacy-breach-generic.
---
 doc/source/_templates/layout.html | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/doc/source/_templates/layout.html b/doc/source/_templates/layout.html
index f47665b..03b028f 100644
--- a/doc/source/_templates/layout.html
+++ b/doc/source/_templates/layout.html
@@ -2,14 +2,4 @@
 
 {%- block extrahead %}
 {{ super() }}
-<!-- Global site tag (gtag.js) - Google Analytics -->
-<script async src="https://www.googletagmanager.com/gtag/js?id=UA-111342564-1"></script>
-<script>
-  window.dataLayer = window.dataLayer || [];
-  function gtag(){dataLayer.push(arguments);}
-  gtag('js', new Date());
-
-  gtag('config', 'UA-111342564-1');
-</script>
-
 {% endblock %}
